apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/IRBuilder.kt

```diff
@@ -1,19 +1,19 @@
-package com.apollographql.apollo.compiler.frontend.ir
+package com.apollographql.apollo.compiler
 
 import com.apollographql.apollo.compiler.ApolloMetadata
 import com.apollographql.apollo.compiler.PackageNameProvider
+import com.apollographql.apollo.compiler.backend.ir.BackendIrBuilder
 import com.apollographql.apollo.compiler.frontend.gql.GQLDocument
 import com.apollographql.apollo.compiler.frontend.gql.GQLEnumTypeDefinition
 import com.apollographql.apollo.compiler.frontend.gql.GQLFragmentDefinition
 import com.apollographql.apollo.compiler.frontend.gql.GQLInputObjectTypeDefinition
 import com.apollographql.apollo.compiler.frontend.gql.GQLOperationDefinition
 import com.apollographql.apollo.compiler.frontend.gql.GQLScalarTypeDefinition
 import com.apollographql.apollo.compiler.frontend.gql.Schema
-import com.apollographql.apollo.compiler.frontend.gql.toIR
 import com.apollographql.apollo.compiler.frontend.gql.usedTypeNames
 import com.apollographql.apollo.compiler.frontend.gql.withTypenameWhenNeeded
 
-class IRBuilder(private val schema: Schema,
+internal class IRBuilder(private val schema: Schema,
```

Comment:
If we rename `BackendIrBuilder.BackendIrBuilderInput` to `FrontedIR` we can rename this builder to `FrontendIRBuilder`